docs/advanced/ObjectSpreadOperator.md

```diff
@@ -0,0 +1,37 @@
+# Object Spread Operator
+
+Since one of the core tenants of Redux is to never mutate state, you'll often find yourself using `Object.assign` to create
+copies of objects with new or updated values. For example, in the `todoApp` below `Object.assign` is used to return a new
+`state` object with an updated `visibilityFilter` property:
+
+```js
+function todoApp(state = initialState, action) {
+  switch (action.type) {
+    case SET_VISIBILITY_FILTER:
+      return Object.assign({}, state, {
+        visibilityFilter: action.filter
+      })
+    default:
+      return state
+  }
+}
+```
+
+While effective, using `Object.assign` can quickly make simple reducers difficult to read given it's rather verbose syntax.
+
+An alternative approach is to use the [object spread syntax](https://github.com/sebmarkbage/ecmascript-rest-spread) proposed for ES7
```

Comment:
Let’s remove “for ES7” because 

a) there won’t be ES7, it’s ES2016
b) it’s already finalized and only includes two tiny features

It’s better to phrase it as “proposed for the next versions of JavaScript”.